snippets/functions/boilerplate/generalFunction.js

```diff
@@ -0,0 +1,42 @@
+exports = async function (arg) {
+  // This default function will get a value and find a document in MongoDB
+  // To see plenty more examples of what you can do with functions see:
+  // https://www.mongodb.com/docs/atlas/app-services/functions/
+
+  // Find the name of the MongoDB service you want to use (see "Linked Data Sources" tab)
+  var serviceName = "mongodb-atlas";
+
+  // Update these to reflect your db/collection
+  var dbName = "db_name";
+  var collName = "coll_name";
+
+  // Get a collection from the context
+  var collection = context.services
+    .get(serviceName)
+    .db(dbName)
+    .collection(collName);
+
+  var findResult;
+  try {
+    // Get a value from the context (see "Values" tab)
+    // Update this to reflect your value's name.
+    var valueName = "value_name";
+    var value = context.values.get(valueName);
+
+    // Execute a FindOne in MongoDB
+    findResult = await collection.findOne({
+      owner_id: context.user.id,
+      fieldName: value,
+      argField: arg,
+    });
+  } catch (err) {
+    console.log("Error occurred while executing findOne:", err.message);
+
+    return { error: err.message };
+  }
+
+  // To call other named functions:
+  // var result = context.functions.execute("function_name", arg1, arg2);
+
+  return { result: findResult };
+};
```

Comment:
We should have done this to begin with, looking good otherwise

```suggestion
exports = async function (arg) {
  // This default function will get a value and find a document in MongoDB
  // To see plenty more examples of what you can do with functions see:
  // https://www.mongodb.com/docs/atlas/app-services/functions/

  // Find the name of the MongoDB service you want to use (see "Linked Data Sources" tab)
  var serviceName = "mongodb-atlas";

  // Update these to reflect your db/collection
  const dbName = "db_name";
  const collName = "coll_name";

  // Get a collection from the context
  const collection = context.services
    .get(serviceName)
    .db(dbName)
    .collection(collName);

  let findResult;
  try {
    // Get a value from the context (see "Values" tab)
    // Update this to reflect your value's name.
    const valueName = "value_name";
    const value = context.values.get(valueName);

    // Execute a FindOne in MongoDB
    findResult = await collection.findOne({
      owner_id: context.user.id,
      fieldName: value,
      argField: arg,
    });
  } catch (err) {
    console.log("Error occurred while executing findOne:", err.message);

    return { error: err.message };
  }

  // To call other named functions:
  // const result = context.functions.execute("function_name", arg1, arg2);

  return { result: findResult };
};
```